ui/admin/app/controllers/application.js

```diff
@@ -105,4 +106,19 @@ export default class ApplicationController extends Controller {
       this.features.enable(feature);
     }
   }
+  /**
+   * Add custom route change validation to prevent refocus when
+   * user is trying to type in search box.
+   * @param {object} transition
+   * @returns
+   */
+  customRouteChangeValidator(transition) {
+    if (
+      transition.to?.name === transition.from?.name &&
+      typeof transition.to.queryParams.search === 'string'
```

Comment:
Hmm is the intention to only have it refocus on a page change or anything that's not search? I can imagine it being still jarring if you're a keyboard user and having your focus being reset just for using a filter and having to go through the page again to goto the next filter

Reply:
Yup that is correct.
Those are good points I'll modify this condition to be less strict